sdks/cpp/CMakeLists.txt

```diff
@@ -20,38 +20,55 @@ cmake_minimum_required (VERSION 3.13.0)
 # For *nix: cmake --build . --target install -- -s
 option(AGONES_SILENT_OUTPUT "Show only warnings/error messages" OFF)
 if (AGONES_SILENT_OUTPUT)
-  function(message)
-    list(GET ARGV 0 MessageType)
-    if(MessageType STREQUAL FATAL_ERROR OR
-      MessageType STREQUAL SEND_ERROR OR
-      MessageType STREQUAL WARNING OR
-      MessageType STREQUAL AUTHOR_WARNING)
-      list(REMOVE_AT ARGV 0)
-      _message(${MessageType} "${ARGV}")
-    endif()
-  endfunction()  
-  
-  set(CMAKE_INSTALL_MESSAGE NEVER)
-  set(CMAKE_VERBOSE_MAKEFILE OFF)
-  set_property(GLOBAL PROPERTY RULE_MESSAGES OFF)
-  set_property(GLOBAL PROPERTY TARGET_MESSAGES OFF)
+    function(message)
+        list(GET ARGV 0 MessageType)
+        list(REMOVE_AT ARGV 0)
+        if (MessageType STREQUAL FATAL_ERROR OR
+            MessageType STREQUAL SEND_ERROR OR
+            MessageType STREQUAL WARNING OR
+            MessageType STREQUAL AUTHOR_WARNING OR
+            NOT ${AGONES_SILENT_OUTPUT}
+            )
+            _message(${MessageType} "${ARGV}")
+        endif()
+    endfunction()
+
+    set(CMAKE_INSTALL_MESSAGE NEVER)
+    set(CMAKE_VERBOSE_MAKEFILE OFF)
+    set_property(GLOBAL PROPERTY RULE_MESSAGES OFF)
+    set_property(GLOBAL PROPERTY TARGET_MESSAGES OFF)
 endif(AGONES_SILENT_OUTPUT)
 
 # Project AGONES
-project(agones VERSION 0.9.0 HOMEPAGE_URL https://github.com/GoogleCloudPlatform/agones LANGUAGES C CXX)
+project(agones VERSION 0.11.0 HOMEPAGE_URL https://github.com/GoogleCloudPlatform/agones LANGUAGES C CXX)
```

Comment:
You should probably plan to write this version based on git tag, since its easy to forget when version number is in cmake file

Reply:
Done